### DISCUSSION TITLE
<How to get clear Excel image in Md language (Github)>

### DISCUSSION THREAD

**Discussion by FNajmudeen** [community/community#67442] [Programming Help]
### Select Topic Area

Show & Tell

### Body

Greetings,

We are facing an issue on getting the Excel image clear to our project. Kindly provide us with some guidance on how we can fix this issue.Image link attached for your kind reference. 

[https://github.com/cquict/coit20246y23t2-project-syd-fathima-musrat/blob/main/images/Task4.2.1AWSCloudPricingEsitimation.png](url)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @FNajmudeen ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion. Though we recommend updating your post to include more than a link to download a file as that may discourage folks from assisting. 

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
